android-components/build.gradle

```diff
@@ -360,6 +360,7 @@ tasks.register("ktlintFormat", JavaExec) {
     classpath = configurations.ktlint
     main = "com.pinterest.ktlint.Main"
     args "-F", "components/**/*.kt" , "samples/**/*.kt", "!**/build/**/*.kt", "buildSrc/**/*.kt", "--baseline=ktlint-baseline.xml"
+    jvmArgs("--add-opens", "java.base/java.lang=ALL-UNNAMED")
```

Comment:
❓ Is this the same as adding `jvmArgs "--add-opens=java.base/java.lang=ALL-UNNAMED"`

If it does require the change, maybe we should also update to use `mainClass` as `main` is deprecated

Reply:
Yes, it is the same.

As for the deprecated JavaExec.main property, I think we can keep it until we upgrade to Gradle 8 ( it will be completely removed in that release) or do it in a separate PR, as it does not pertain to the issue in case.